### PR TITLE
hinge new recipe

### PR DESCRIPTION
**Pull request by mictadlo**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/hinge/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  skip: True  # [not py27]
+
+source:
+  fn: {{ name|lower }}_{{ version }}.tar.gz
+  url: https://github.com/fxia22/HINGE/archive/v0.41.tar.gz
+  md5: d2012769150da11577c93513ec0db50d
+
+requirements:
+  build:
+  - python >=2.7,<3
```
Comment (bgruening) [inline recipes/hinge/meta.yaml]: You don't need this if you have the skip statement.

File: recipes/hinge/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  - boost
+  - hdf5
+  - numpy
+  - ujson
+  - colormap
+  - easydev
+  
+
+  run:
+  - python >=2.7,<3
```
Comment (bgruening) [inline recipes/hinge/meta.yaml]: same here

File: recipes/hinge/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  - libgcc # [not osx]
+  - boost
```
Comment (bgruening) [inline recipes/hinge/meta.yaml]: please pin the boost library as specified in our guidelines.

File: recipes/hinge/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  - dextractor
+
+
+test:
+  commands:
+    - hinge -h #| grep "" > /dev/null
+
+about:
+  home: https://github.com/yechengxi/SparseAssembler
+  summary: HINGE is a long read assembler based on an idea called hinging
```
Comment (bgruening) [inline recipes/hinge/meta.yaml]: can you also add `licence` please

**Comment by druvus** [comment]
You also need to add spdlog asa dependency. See the submodules file. I have recipe for it as a PR but never merged it.
@bioconda/core how should we test header only recipes? Or is it better to add spdlog to conda-forge?

**Comment by bgruening** [comment]
@druvus I think conda-forge would be good. conda-forge has also a lot of example how to test header only ones. You could for example write a tiny sample code and compile it.

**Comment by mictadlo** [comment]
I changed `mata.yaml`, but the build failed and I can't figure out why. What did I miss or is it just because I miss spdlog?

**Comment by druvus** [comment]
I just did a PR to add spdlog to conda-forge so it would hopefully be available soon.   

You would then need patch to look for systemwide spdlog and not relative path.

**Comment by mictadlo** [comment]
@druvus any updates on spdlog?

**Comment by druvus** [comment]
@mictadlo You can follow the progress here conda-forge/staged-recipes#2268. The recipe pass all tests but it is not merged yet.

**Comment by druvus** [comment]
spdlog just got merged at conda-forge

**Comment by johanneskoester** [comment]
We have migrated our build system from Travis CI to Circle CI. If you would like to continue working on this PR, please update with latest upstream master and reopen this PR.

**Pull request closed without merging**